pkg/podexecutor/staticpod.go

```diff
@@ -107,9 +118,9 @@ func (s *StaticPodConfig) APIServer(ctx context.Context, etcdReady <-chan struct
 			Command:   "kube-apiserver",
 			Args:      args,
 			Image:     s.Images.KubeAPIServer,
-			Dirs:      ssldirs,
+			Dirs:      append(ssldirs, filepath.Dir(auditLogFile)),
```

Comment:
Don't we need to mount the audit policy file into the pod as well as the log file?

Reply:
nah, the code automatically mount all argument files https://github.com/rancher/rke2/blob/aeb2f9e07e2cb622380b02a360b771bc31941dc9/pkg/staticpod/staticpod.go#L229

Reply:
Ah, I missed that bit of magic. Cool!